qcengine/programs/entos.py

```diff
@@ -181,68 +194,59 @@ def build_input(
+                "xtb": {"charge": input_model.molecule.molecular_charge,},
             }
 
             # Resolve keywords (extra options) for the energy command
             caseless_keywords = {k.lower(): v for k, v in input_model.keywords.items()}
-            energy_extra_options = {}
+            energy_keywords_extra = {}
             for key in caseless_keywords.keys():
-                if key in self._keyword_map[energy_command]:
-                    energy_extra_options[key] = caseless_keywords[key]
+                if key in self._energy_commands[energy_command]:
+                    energy_keywords_extra[key] = caseless_keywords[key]
 
             # Additional sub trees
             structure = {"structure": {"file": "geometry.xyz"}}  # Structure sub tree
-            print_results = {"print": {"results": True}}  # Print sub tree
-            name_results = {"name": "json_results"}
 
             # Create the input dictionary for a energy call
             if input_model.driver == "energy":
                 input_dict = {
                     energy_command: {
                         **structure,
-                        **energy_options[energy_command],
-                        **energy_extra_options,
-                        **name_results,
+                        **energy_keywords_from_input_model[energy_command],
+                        **energy_keywords_extra,
                     },
-                    **print_results,
                 }
             # Create the input dictionary for a gradient call
             elif input_model.driver == "gradient":
                 input_dict = {
                     "gradient": {
                         **structure,
-                        energy_command: {**energy_options[energy_command], **energy_extra_options},
-                        **name_results,
+                        energy_command: {**energy_keywords_from_input_model[energy_command], **energy_keywords_extra,},
                     },
-                    **print_results,
                 }
-            # TODO Add support for hessians
-            elif input_model.driver == 'hessian':
+            elif input_model.driver == "hessian":
                 input_dict = {
                     "hessian": {
                         **structure,
-                        energy_command: {**energy_options[energy_command], **energy_extra_options},
-                        **name_results,
+                        energy_command: {**energy_keywords_from_input_model[energy_command], **energy_keywords_extra},
                     },
-                    **print_results,
                 }
             else:
                 raise NotImplementedError(f"Driver {input_model.driver} not implemented for entos.")
 
             # Write input file
-            input_file = self.write_input_recursive(input_dict)
+            input_file = [
+                f"entos_policy(version = '{self.get_version()}')",
```

Comment:
It looks like CI is failing when I added this line to the entos input file.  @mattwelborn do you think the entos_policy is worth keeping? If so I'll just add the `@using_entos` mark to the `test_entos_input_formatter`.

Reply:
I added the @using_entos.